schema.graphql

```diff
@@ -2194,3 +2194,102 @@ type ChildIdentity @entity {
   createdBlock: Block!
   updatedBlock: Block!
 }
+
+"""
+Represents signer types
+"""
+enum SignerTypeEnum {
+  Account
+  Identity
+}
+
+"""
+Represents a MultiSig
+"""
+type MultiSig @entity {
+  id: ID!
+  address: String! @index(unique: true)
+  creator: Identity!
+  creatorAccount: Account!
+  signers: [MultiSigSigner]! @derivedFrom(field: "multisig")
+  proposals: [MultiSigProposal]! @derivedFrom(field: "multisig")
+  signaturesRequired: Int!
+  createdBlock: Block!
+  updatedBlock: Block!
+}
+
+"""
+Represents MultiSig signer status
+"""
+enum MultiSigSignerStatusEnum {
+  Authorized ## when a signer is added
```

Comment:
(nit) I think `Pending` is a bit more clear. `Authorized` makes it seem like the account is already added to me. Not a big deal either way.

Reply:
I wanted to keep it as similar as the chain